source/trackers.json

```diff
@@ -727,6 +818,21 @@
         "riot.im": "element",
         "graph.facebook.com": "facebook_audience",
         "app-measurement.com": "firebase",
+        "fcm.googleapis.com": "firebase",
+        "firebaseappcheck.googleapis.com": "firebase",
+        "firebaseapp.com": "firebase",
+        "firebase.com": "firebase",
+        "firebasedynamiclinks.googleapis.com": "firebase",
```

Comment:
Where is it used? I need a real example of use.

Reply:
It is not worth adding domains that are not in use. Because of this, the database will contain a lot of irrelevant information.

Reply:
I am aware of most of these domains.